app/configurations/config.okc.js

```diff
@@ -68,6 +68,16 @@ export default configMerger(walttiConfig, {
   showVehiclesOnSummaryPage: true,
   showWeatherInformation: false,
 
+  defaultSettings: {
+    bikeSpeed: 5.59,
+    showBikeAndParkItineraries: false,
+  },
+
+  defaultOptions: {
```

Comment:
I'd like to add a commit that these are converted from km/h, and *not* carefully chosen to be these specific values.

```suggestion
  defaultOptions: {
    // converted from km/h
```

Reply:
+1, added a comment with the used speeds in mph